accord-core/src/main/java/accord/local/Node.java

```diff
@@ -246,11 +261,8 @@ public void withEpoch(long epoch, Runnable runnable)
         }
         else
         {
-            CommandStore executingOn = CommandStore.maybeCurrent();
             configService.fetchTopologyForEpoch(epoch);
-            AsyncChain<Void> chain = topology.awaitEpoch(epoch);
-            chain = (executingOn == null ? chain.addCallback(runnable) : chain.addCallback(runnable, executingOn));
-            chain.begin(agent());
+            topology.awaitEpoch(epoch).addCallback(runnable).begin(agent);
```

Comment:
`topology.awaitEpoch` already attaches the `CommandStore` so its wasteful to do it again